src/styles/blocks/filters.scss

```diff
@@ -0,0 +1,28 @@
+.filters {
+  position: absolute;
+  right: 0;
+  left: 0;
+
+  @media (max-width: 430px) {
+    bottom: 10px;
+  }
+
+  &__link {
+    margin: 3px;
+    padding: 3px 7px;
+
+    border: 1px solid transparent;
+    border-radius: 3px;
+
+    color: inherit;
+    text-decoration: none;
+
+    &:hover {
+      border-color: rgba(175, 47, 47, 0.1);
+    }
+
+    &.selected {
+      border-color: rgba(175, 47, 47, 0.2);
```

Comment:
It is recommended to use variables for colors (e.g., $border-selected-color) to maintain a consistent color scheme throughout the project.

Reply:
I created variables for all colors.